spec/controllers/api/v1/organizations_controller_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'spec_helper'
+
+describe Api::V1::OrganizationsController, type: :controller do
+  let(:authorized_user) { create(:user) }
+  let(:organization) { build(:organization, listed_at: Time.now, owner: authorized_user) }
+  let(:unlisted_organization) { build(:organization, listed_at: nil) }
+
+  let(:scopes) { %w(public organization) }
+
+  describe "when not logged in" do
+    describe "#index" do
+
+      it 'returns only the listed organization' do
```

Comment:
i wondered if we should be testing that a non-logged in can't access non-public resources? You may have covered this with the unauthorized user though.